cloud-build/README.MD

```diff
@@ -0,0 +1,12 @@
+# Continuous Integration via Cloud Build
+
+Cloud Build is configured in our GCP project (google.com:cloud-dataproc-ci) to trigger whenever there are PRs from a fork to the main repository. We are using [GitHub App Triggers Alpha](http://go/gcb-github#github-app-triggers) to integrate with GitHub.
```

Comment:
I don't think we want to expose the GCP project name publicly.

Reply:
Removed.